src/main/java/seedu/taassist/model/ModelManager.java

```diff
@@ -133,6 +134,13 @@ public void deleteModuleClass(ModuleClass target) {
         taAssist.removeModuleClass(target);
     }
 
+    @Override
+    public void deleteModuleClasses(Collection<ModuleClass> moduleClasses) {
+        for (ModuleClass moduleClass : moduleClasses) {
```

Comment:
`null` check is missing. `moduleClasses` as well as each `moduleClass` need not be `null`.